src/RoleObjects/AccessibilityObject.js

```diff
@@ -151,23 +150,141 @@ export default class AccessibilityObject {
    * @access public
    * @returns {String} string containing the letter to use for the keyboard shortcut
    */
   get accessKey() {
     return this._reactProps.accessKey;
   }
 
+
+  /**
+   * Sets the keyboard shortcut to click the DisplayObject
+   * @access public
+   * @param {String} str - "yes" or "no" Specifies that the content of the element should be translated or not
+   */
+  set translate(str) {
+    this._reactProps.translate = str;
+  }
+
+  /**
+   * Retrieves the keyboard shortcut to click the DisplayObject
+   * @access public
+   * @returns {String} string containing the letter to use for the keyboard shortcut
+   */
+  get translate() {
+    return this._reactProps.translate;
+  }
+
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @param { boolean } bool
+   */
+  set spellcheck(bool) {
+    this._reactProps.spellCheck = bool;
+  }
+
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @returns {boolean} boolean: true or false
+   */
+  get spellcheck() {
+    return this._reactProps.spellCheck;
+  }
+
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @param { boolean } bool true or false
+   */
+  set contenteditable(bool) {
+    this._reactProps.contentEditable = bool;
+  }
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @returns {boolean} true or false
+   */
+  get contenteditable() {
+    return this._reactProps.contentEditable;
+  }
+
+
+  /**
+   * Sets the title for the DisplayObject
+   * @access public
+   * @param {String} title - string mouse hover show text of title as tooltip
+   */
+  set title(title) {
+    this._reactProps.title = title;
+  }
+
+  /**
+   * Retrieves the title for the DisplayObject
+   * @access public
+   * @returns {String} string mouse hover show text of title as tooltip
+   */
+  get title() {
+    return this._reactProps.title;
+  }
+
+  /**
+   * The lang attribute specifies the language of the element's content.
+   * @access public
+   * @returns {String} lang: eg. "en" for English, "es" for Spanish, "fr" for French, and so on.
+   */
+  get lang() {
+    return this._reactProps.lang;
+  }
+
+  /**
+   * The lang attribute specifies the language of the element's content.
+   * @access public
+   * @param {String} lang: eg. "en" for English, "es" for Spanish, "fr" for French, and so on.
+   */
+  set lang(lang) {
+    this._reactProps.lang = lang;
+  }
+
+  /**
+   * The dir attribute specifies the text direction of the element's content.
+   * @access public
+   * @param {String} direction
+   * ltr: Left-to-right text direction, rtl: Right-to-left text direction
```

Comment:
Nit: to help make the different valid values clearer, please put rtl on its own line.

Reply:
Done